server/index.js

```diff
@@ -4,13 +4,41 @@ const bodyParser = require('body-parser')
 const dbFunctions = require('./db.js')
 const levelup = require('levelup')
 const db = levelup('./hint-hint')
+const moment = require('moment')
+const schedule = require('node-schedule')
+const twilio = require('twilio')
+const dotenv = require('dotenv').config()
+const client = new twilio(process.env.TWILIO_ACCOUNT_SID, process.env.TWILIO_AUTH_TOKEN)
 const app = express()
 
 app.use(bodyParser.json())
 
 const publicPath = path.join(__dirname, '/public')
 app.use(express.static(publicPath))
 
+schedule.scheduleJob('*/10 * * * *', () => {
+  const now = Date.now()
+  dbFunctions.getCollection(db, 'events')
+    .then((data) => {
+      for (let i = 0; i < data.length; i++) {
+        let value = data[i]
+        const time = moment(value.date).format('hh:mm a')
+        if (Math.abs(now - value.date) <= 3600000 && (now - value.date) < 0 && value.notified !== true) {
```

Comment:
you can clean this up a bit by first obtaining the items that are candidates for notifications, using `Array.filter`. Also, `client.messages.create(..)` returns a promise, so we can't call `dbFunctions.updateCollection` until all of those promises are done. You can leverage `Promise.all` here.